src/metal/metal-texture.cpp

```
@@ -221,6 +222,38 @@ Result DeviceImpl::createTexture(const TextureDesc& desc_, const SubresourceData
     return SLANG_OK;
 }
 
+Result DeviceImpl::createTextureFromNativeHandle(NativeHandle handle, const TextureDesc& desc, ITexture** outTexture)
+{
+    AUTORELEASEPOOL
+
+    if (!outTexture)
```

Cppcheck (2.20.0)

[style] The function 'rbegin' is never used.

(unusedFunction)